onnx/defs/nn/defs.cc

```diff
@@ -227,11 +227,11 @@ std::function<void(OpSchema&)> PoolOpSchemaGenerator(
  subset of the input tensor according to the kernel size and downsampling the
  data into the output tensor Y for further processing. The output spatial shape will be following:
  ```
- output_spatial_shape[i] = floor((input_spatial_shape[i] + pad_shape[i] - {kernelSpatialShape}) / strides_spatial_shape[i] + 1)
+ output_spatial_shape[i] = floor((input_spatial_shape[i] + pad_shape[i] - {kernelSpatialShape}) / strides_spatial_shape[i] - 1)
```

Comment:
Doesn't sound right. Consider the simple case where there is no padding, stride is 1, and kernel-size is same as input-size. We expect the output size to be 1, right? So, the original "+1" sounds right, not "-1". Can you double-check? Thanks!

Reply:
I responded to the original issue, please take a look.